src/test/java/com/wikia/webdriver/testcases/searchtests/SearchPageTests.java

```diff
@@ -0,0 +1,23 @@
+package com.wikia.webdriver.testcases.searchtests;
+
+import com.wikia.webdriver.common.core.annotations.Execute;
+import com.wikia.webdriver.common.core.annotations.InBrowser;
+import com.wikia.webdriver.common.core.drivers.Browser;
+import com.wikia.webdriver.common.templates.NewTestTemplate;
+import com.wikia.webdriver.elements.oasis.pages.SearchPage;
+
+import org.testng.annotations.Test;
+
+@InBrowser(browser = Browser.CHROME)
+@Execute(onWikia = "mercuryautomationtesting")
+public class SearchPageTests extends NewTestTemplate {
+
+  @Test(groups = "searchPageTests_exactMatchShowsRelatedCommunityModule")
+  public void searchPageTests_exactMatchShowsRelatedCommunityModule() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS00100)